graphql_api/types/account/account.py

```diff
@@ -29,3 +37,20 @@ def resolve_total_seat_count(account: Account, info: GraphQLResolveInfo) -> int:
 @sync_to_async
 def resolve_activated_user_count(account: Account, info: GraphQLResolveInfo) -> int:
     return account.activated_user_count
+
+
+@account_bindable.field("organizations")
+@convert_kwargs_to_snake_case
+def resolve_organizations(
+    account: Account,
+    info: GraphQLResolveInfo,
+    ordering=AccountOrganizationOrdering.ACTIVATED_USERS,
```

Comment:
just to confirm, default ordering is by plan_activated_users in descending order